.github/workflows/test-build-deploy.yml

```diff
@@ -96,13 +110,15 @@ jobs:
 
```

Comment:
Do we need
`          docker build -t $REGISTRY/$REPOSITORY:$IMAGE_TAG .`

In the build step above?

Reply:
The build is done in the `build` step "app" is the image, this is still inscope at this point. You can apply tags to images after they are built with `docker tag <image> <tags>` which is what I have done in the push section

Reply:
Aha, gotcha 👍 Thank you